src/dojos/cd-dojo-details.vue

```diff
@@ -109,6 +109,7 @@
       return {
         dojoDetails: {},
         user: {},
+        googleMapsApiKey: process.env.GOOGLE_MAPS_API_KEY,
```

Comment:
expose it through vue.config instead.
It's fine in main.js (entrypoint), but i'd avoid having process.env in the rest of the code